### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sidujjain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,29 +49,33 @@ If you want to contribute to **jovian**(Python package), review the [**Contribut
...
-        <th>jovian(Python Package)</th>
+        <th>Jovian(Python Package)</th>
         <th>Jovian(Webapp)</th>
     </tr>
     <tr>
         <td>Issues or Feature Request</td>
-        <td align="center"> <a href="https://github.com/JovianML/jovian-py/issues">Forum</a> </td>
-        <td align="center"> <a href="https://github.com/JovianML/jovian-support/issues">Forum</a> </td>
+        <td align="center"> <a href="https://github.com/JovianML/jovian-py/issues" target="_blank">Forum</a> </td>
+        <td align="center"> <a href="https://jovian.ml/forum/c/help/22" target="_blank">Forum</a> </td>
```
Comment (PrajwalPrashanth) [inline README.md:58]: Seems odd, these links are still opening up in the same tab, can you verify once https://github.com/JovianML/jovian-py/tree/sid/update-readme

File: README.md [hunk 2]
```diff
@@ -49,29 +49,33 @@ If you want to contribute to **jovian**(Python package), review the [**Contribut
...
-<br>
+<br/>
 Twitter : [@JovianML](https://twitter.com/JovianML)
-<br>
+<br/>
 Slack : [Slack Invite](https://bit.ly/jovian-users)
+<br/>
+YouTube: [YouTube Channel](https://www.youtube.com/jovianml)
+<br/>
+LinkedIn: [LinkedIn Page](https://www.linkedin.com/company/jovianml/)
```
Comment (PrajwalPrashanth) [inline README.md:81]: We can add link to forums here.

File: README.md (hunk 1 above)
Comment (sidujjain) [reply README.md:58]: Seems like Github markdown rendering doesn't support `target="_blank"`

File: README.md (hunk 2 above)
Comment (sidujjain) [inline README.md:81]:
```suggestion
LinkedIn: [LinkedIn Page](https://www.linkedin.com/company/jovianml/)
<br/>
Forum: [Data Science Forum](https://jovian.ml/forum/)
```

**Pull request merged**